scripts/README.txt

```diff
@@ -0,0 +1,118 @@
+## Flow_stats's scripts
+
+This folder contains Flow_stats's related scripts.
+
+### Integration Kytos-ng and Zabbix for Monitoring
+
+[`kytos_zabbix.py`](./kytos_zabbix.py) is a script to help on the Kytos-ng monitoring for Zabbix.
+
+#### Pre-requisites
+
+- There's no additional Python libraries dependencies required (the libs we use are usually installed by default).
+- Make sure your Kytos server is running and allowing requests on the Napp `flow_stats`, `mef_eline` and `topology` (via HTTP GET method).
+- When using authentication (recommended), you will have three options to provide the credentials: via command line parameters (unsafe) `--user` and `--pass`; via a file in the filesystem (`--passfile`) which contains the username in the first line and the password in the second line (you can change the permissions of the file to restrict access); via environment variables, such as:
+- Export MongoDB related variables that [db/client.py](../db/client.py) uses, make sure the hosts names can be resolved:
+
+```
+export KYTOS_URL=https://mykytoserver.domain.com/api
+export KYTOS_USERNAME=zabbix_reader
+export KYTOS_PASSWORD=changeme123
+```
+
+#### How to use
+
+The `kytos_zabbix.py` script has a few monitoring capabilities, such as the monitoring option `-o`: 1 - status of switches; 2 - status of links; 3 - status of EVCs; 4 - statistics of EVCs. You can filter by the target to be monitored by using the option `-t` (target), example: to get the status of switch01 (dpid 00:00:00:00:00:00:00:01) one would use `-o 1 -t 00:00:00:00:00:00:00:01`. Furthermore, when collecting statistics for EVCs, you can filter by (option `-s`): 1 - bytes/UNI_A, 2 - bytes/UNI_Z , 3 - packets/UNI_A, 4 - packets/UNI_Z.
+
+Here is the complete help and options:
+
+```
+# /usr/share/zabbix/externalscripts/kytos_zabbix.py --help
```

Comment:
Italo, could you find and replace the path accordingly `/usr/share/zabbix/externalscripts/kytos_zabbix.py`, or just a local relative folder like `./kytos_zabbix.py --help`?

Reply:
Fixed in b7072ac